src/main/kotlin/uk/gov/justice/digital/hmpps/prisonestate/resource/PrisonResource.kt

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.digital.hmpps.prisonestate.resource
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL
+import io.swagger.annotations.*
+import org.springframework.http.MediaType
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.prisonestate.ErrorResponse
+import uk.gov.justice.digital.hmpps.prisonestate.jpa.Prison
+import uk.gov.justice.digital.hmpps.prisonestate.service.PrisonService
+import javax.validation.constraints.Size
+
+@RestController
+@Validated
+@RequestMapping("/prisons", produces = [MediaType.APPLICATION_JSON_VALUE])
+class PrisonResource(private val prisonService: PrisonService) {
+  @GetMapping("/id/{prison_id}")
+  @ApiOperation("Get specified prison")
+  @ApiResponses(value = [
+    ApiResponse(code = 400, message = "Bad request.  Wrong format for prison_id.", response = ErrorResponse::class),
```

Comment:
I've only just realised that I don't need to add the 200 case, it will add by default